com.eclipsesource.modelserver.emf/src/main/java/com/eclipsesource/modelserver/emf/common/JsonResponse.java

```diff
@@ -21,41 +21,69 @@
 import com.fasterxml.jackson.databind.node.ObjectNode;
 import org.jetbrains.annotations.Nullable;
 
-public class JsonResponse {
+/**
+ * Structure of JsonResponse
+ * 
+ * {
+ *   "type": "success" | "error" | "fullUpdate" | "incrementalUpdate" | "dirtyState"
+ *   "data": String | Boolean | EObject
+ * }
+ */
 
-	public static ObjectNode success() {
+public class JsonResponse {
+	
+	private static ObjectNode type(String type) {
+		// Allowed types: success, error, fullUpdate, incrementalUpdate
+		return Json.object(Json.prop("type", Json.text(type)));
```

Comment:
We should think about introducing an enum for the different `type`s. If you agree, let's create a followup issue.

Reply:
Yes I agree, will create an issue for that

Reply:
Issue #107